sys/fido2/ctap/ctap_mem.c

```diff
@@ -11,75 +11,47 @@
  * @{
  * @file
  *
- * @author      Nils Ollrogge <nils.ollrogge@fu-berlin.de>
+ * @author      Nils Ollrogge <nils-ollrogge@outlook.de>
  * @}
  */
-
-#include <string.h>
-
-#include "bitarithm.h"
-
-#include "mtd.h"
 #include "mtd_flashpage.h"
 
 #include "fido2/ctap/ctap_mem.h"
 #include "fido2/ctap/ctap_utils.h"
 
-#define ENABLE_DEBUG    (0)
+#define ENABLE_DEBUG 0
 #include "debug.h"
 
+static mtd_dev_t *_mtd_dev;
```

Comment:
```suggestion
static mtd_dev_t *_mtd_dev = CONFIG_FIDO2_MTD_DEV;
```

We should make the application select this

Reply:
Doing this I get:
```bash
/home/h0ps/Programming/RIOT/tests/sys/fido2_ctap_hid/bin/nrf52840dongle/riotbuild/riotbuild.h:21:30: error: initializer element is not constant
   21 | #define CONFIG_FIDO2_MTD_DEV mtd_aux
      |                              ^~~~~~~
```

Therefore I am setting `_mtd_dev` inside `fido2_ctap_mem_init`